Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3
+RUN apt-get update \
+    && apt-get install -y postgresql-client \
```

Comment:
Why would you need postgresql-client?

Reply:
Python needs it to communicate with Postgresql